cmd/experimental/kueue-viz/INSTALL.md

```diff
@@ -0,0 +1,107 @@
+# Installation with helm
+
+KueueViz can be installed using helm using the following command and 
+by ensuring that `enableKueueViz` is set to `true`:
+
+```
+KUEUE_VERSION=v0.11.0
+helm upgrade --install kueue oci://us-central1-docker.pkg.dev/k8s-staging-images/kueue/charts/kueue \
```

Comment:
Our docs should not include the staging repo.

we publish these images to registry.k8s on a release. We should nudge users to use permanent images.

I believe these staging images have a short life.

Reply:
will update that